apps/coscrad-frontend/src/components/resource-info/presenters/resource-info.presenter.tsx

```diff
@@ -14,7 +15,7 @@ export const ResourceInfoPresenter = ({
  */
 IAggregateInfo<ResourceType>): JSX.Element => (
     <Link to={`/${routes.resources.ofType(resourceType).index}`}>
-        <Card>
+        <PrimaryCard>
```

Comment:
One thing that is really interesting is that styled-components involves wrapping, whereas `scss modules` involves injecting props. I wonder if it's possible to create a pattern where scss modules become wrappers?